docs/src/main/sphinx/object-storage/file-system-cache.md

```diff
@@ -134,6 +134,17 @@ enable and configure caching for the specific catalogs.
     smaller values are less efficient since they result in more individual downloads.
 :::
 
+## Monitoring
+
+The cache exposes the
+[Alluxio JMX client metrics](https://docs.alluxio.io/ee-da/user/stable/en/reference/Metrics-List.html#client-metrics)
+under the `org.alluxio` package. Not all the JMX metrics on the workers are
+propagated to the coordinator, and are not available in the JMX connector.
+They must be collected on the workers directly.
```

Comment:
I would skip the last 2 lines, that sounds like something to be investigated and fixed rather than being a caveat in docs. We have a product test querying alluxio metrics, so it clearly works in some scenario.

Reply:
Yes, I'm a bit confused why it does not work in all scenarios. One of the metrics missing on the coordinator are `Client.CacheHitRate`, which both of the requests on Slack were asking for. I'll open an issue for looking into this. The metric is available on the workers (maybe it's an issue of the metric never being registered on the coordinator, so it will never be propagated to the coordinator by the mechanism which handles this?).

Reply:
Created https://github.com/trinodb/trino/issues/20794